db/seeds/developers.rb

```diff
@@ -82,3 +82,10 @@
   role_level: RoleLevel.new(RoleLevel::TYPES.map { |t| [t, true] }.to_h)
 })
 developer.update_and_notify(search_status: :not_interested) unless developer.not_interested?
+
+# Feature Update Developer
+developer = SeedsHelper.create_developer!("developer", {
+  hero: "Feature Update developer",
+  location: SeedsHelper.locations[:new_york]
+})
+developer.notify_product_feature_update unless Notification.exists?(type: Developers::ProductFeatureNotification.name)
```

Comment:
We already have a seed for a developer with `developer@exmaple.com` so this is a little redundant. Instead, let's move line 91 to line 75 so it will apply to the "core" developer.

We can also remove the comment.